conftest.py

```diff
@@ -227,9 +227,7 @@ def pytest_collection_modifyitems(session, config, items):  # pylint: disable=un
 
 
 @pytest.hookimpl(hookwrapper=True)
-def pytest_terminal_summary(
-    terminalreporter, exitstatus, config
-):  # pylint: disable=unused-argument
+def pytest_terminal_summary(terminalreporter, exitstatus, config):  # pylint: disable=unused-argument
```

Comment:
ruff ignores `# pylint: disable=unused-argument` when calculating the line length.